src/snowflake/snowpark/session.py

```diff
@@ -739,6 +751,21 @@ def auto_clean_up_temp_table_enabled(self, value: bool) -> None:
                 "value for auto_clean_up_temp_table_enabled must be True or False!"
             )
 
+    @large_query_breakdown_enabled.setter
+    @experimental_parameter(version="1.22.0")
+    def large_query_breakdown_enabled(self, value: bool) -> None:
+        """Set the value for large_query_breakdown_enabled"""
```

Comment:
Can we add docstring for it, just explaining what this optimization is doing and is there any effect. And mention the side effect of issuing an extra query for checking transaction .